### PR TITLE
config for accessControlMaxAge

### PR DESCRIPTION
**Pull request by archeus**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -140,6 +140,12 @@ variable "time_limit_ms" {
   type        = number
 }
 
+variable "cors_access_control_max_age" {
+  description = "The amount of time in seconds that the browser should cache the preflight response"
+  default     = 5
```
Comment (andrein) [inline variables.tf]: According to [the docs](https://docs.snowplow.io/docs/pipeline-components-and-applications/stream-collector/configure/#:~:text=collector.cors.accessControlMaxAge), the default value is 60 minutes.

Comment (archeus) [reply variables.tf]: the [initial terraform version](https://github.com/smartx-net/terraform-aws-collector-kinesis-ec2/commit/a0fc9bed4a3773672cd42f83780524c90844fe44#diff-1ccb1dc63168fbdc08c81ba77138785a54a9084df1bab2f2867c5a276c436375R52) specifically set the value to 5 seconds. Later we have increased it to [5 minutes](https://github.com/smartx-net/terraform-aws-collector-kinesis-ec2/commit/59540147b597b2ed744d2157c3c277952368ce46) but then we realised it's better to preserve the initial terraform default behaviour and just allow the value to be configured.

Comment (andrein) [reply variables.tf]: I'd stick with the upstream default.

Comment (archeus) [reply variables.tf]: updated

**Pull request merged**